experiments.tex

```diff
@@ -84,7 +85,8 @@ \subsection{Effect of $\overline{\tcomp}/\overline{\tIO}$ on Performance}
 \subsubsection{Effect of Increased Compute workload for RMSD Task}
 
 The increased computational workloads corresponded to $\overline{\tcomp}/\overline{\tIO}$ ratios of 11, 19, 27 respectively as shown in Table \ref{tab:load-ratio}.
-In fact, the $\overline{\tcomp}/\overline{\tIO}$ ratio for the higher workloads (with factor $X$) should be $X$ times the $\overline{\tcomp}/\overline{\tIO}$ for the $1\times$ workload (In agreement with theoretical prediction in Table \ref{tab:load-ratio}).
+\gpnote{I am not sure if this breaking passes the same message. It needs fixing.}
+In fact, the $\overline{\tcomp}/\overline{\tIO}$ ratio for the higher workloads (with factor $X$) should be $X$ times the $\overline{\tcomp}/\overline{\tIO}$ for the $1\times$ workload (in agreement with theoretical prediction in Table \ref{tab:load-ratio}).
 This is since on average the I/O workload of each rank is $N_{\text{b}} \times \tIO $, which is independent of $X$. 
```

Comment:
"This is since" is not a good way to start a sentence. I know, it's not yours but maybe change to just continuing the previous sentence and the next one:

     .... \ref{tab:load-ratio}) because the average I/O workload of each rank is $N_{\text{b}} \times \tIO $, which is independent of $X$, and the workload for the computation is $X \times N_{\text{b}} \times \tcomp$, so that the ratio is $X \times \overline{\tcomp}/\overline{\tIO}$.

It's long but at least flows better.

Reply:
I rewrote it again in 78db4ffe23cd66d555c7334237b271705fd245c0